scrapers/ModelMediaUS.yml

```diff
@@ -24,5 +30,28 @@ xPathScrapers:
         postProcess:
           - subScraper:
               selector: //stream/@poster
-
-# Last Updated March 27, 2022
+  altSceneScraper:
+    common:
+      $container: //main[@id='MusContainer']
+    scene:
+      Title: $container//h1
+      Date:
+        selector: //div[contains(.,"Released")]/following-sibling::div/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details:
+        selector: $container//h1/following-sibling::p/text()
+      Performers:
+        Name: 
+          selector: //div[contains(.,"Cast")]/following-sibling::div//a/text()
+      Tags:
+        Name: $container//h1/following-sibling::div//a[contains(@href,"videos?tag")]/span/text()
+      Image:
+        selector: //iframe[@id="video-player"]/@src
+        postProcess:
+          - subScraper:
+              selector: //stream/@poster
```

Comment:
This seems to work for the modelmediaassia site but gets access denied for the other sites?
```yaml
      Image: //div[contains(@class,"mus-z-index-2")]/img[contains(@class,"mus-img-cover")]/@src
```
Seems to work for delphine and jerkaoke
From a quick test it seems modelmediaasia needs to be moved to the `sceneScraper`